src/runtime_src/core/pcie/driver/linux/xocl/subdev/icap.c

```diff
@@ -1580,6 +1580,64 @@ static int icap_create_subdev_cu(struct platform_device *pdev)
 	return err;
 }
 
+// Create subdev for PS kernels
+static int icap_create_subdev_scu(struct platform_device *pdev)
+{
+	struct icap *icap = platform_get_drvdata(pdev);
+	xdev_handle_t xdev = xocl_get_xdev(pdev);
+	struct ps_kernel_node *ps_kernel = icap->ps_kernel;
+	struct ps_kernel_data *scu_data;
+	struct xrt_cu_info info;
+	int err = 0, i, j;
+	int inst = 0;
+
+	/* Let SCU controller know the dynamic resources */
+	for (i = 0; i < ps_kernel->pkn_count; ++i) {
+		scu_data = &ps_kernel->pkn_data[i];
+
+		for (j=0; j < scu_data->pkd_num_instances; ++j) {
+			struct xocl_subdev_info subdev_info = XOCL_DEVINFO_SCU;
```

Comment:
XOCL_DEVINFO_SCU should be something new in devices.h. But I don't see devices.h changes. Maybe it is missed?